src/lib.rs

```diff
@@ -706,12 +721,75 @@ impl Parse for BitsAttr {
     }
 }
 
+/// Endianness determines the layout of the bits:
+///
+/// ```
```

Comment:
This is a lot of documentation for a private type. Instead, we need this documentation at the module level so that `cargo docs` renders it and the user can find it. 

I would suggest a section before "fmt::Debug and Default" titled "Bit Order" with these examples (https://github.com/wrenger/bitfield-struct-rs/pull/19/commits/5281d4b4b0f2c86c119b3561fd49eb9cfeada8ef#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R180).

Reply:
ah, I wasn't even thinking about the visibility here.

moved, and I'll try to get the docs rendering locally as well just to check them out.